### DISCUSSION TITLE
Ability to use incognito

### DISCUSSION THREAD

**Discussion by njhughes-01** [9p4/jellyfin-plugin-sso#103] [Q&A] [answered]
Can we change modify polices like [AmbientAuthenticationInPrivateModesEnabled] so users who happen to use incognito or some browser utilities can still utilize the SSO? Currently, using private browser breaks the login

**Comment by 9p4** [comment, marked as answer]
I haven't run into any issues using the plugin in incognito mode yet. It also seems like ambient authentication setting is a browser-side configuration and nothing I can do on the plugin side.

Let me know if I got anything above wrong. I am definitely interested in your use case and why/how you are using something like Kerberos with the plugin (which is, afaik, what ambient authentication is).

**Comment by njhughes-01** [comment]
So after a few hours of testing, the issues had something to do with my authentik redirects. This was a setup issue code base functions with incognito
